config/jobs/extended-support/extended-support-periodics.yaml

```diff
@@ -27,4 +27,50 @@ periodics:
             memory: "36Gi"
           requests:
             cpu: 4
-            memory: "36Gi"
\ No newline at end of file
+            memory: "36Gi"
+- interval: 1m
```

Comment:
@ArkaSaha30 any reason why we are so aggressive ? the execution time of this job is around 30 min.

Reply:
Yes, sometimes when the job does not wait for autoscaling and fails due to resource crunch we do not want to wait until it can run again. Once we know it had a successful run we will revert the interval to 1h as per upstream.